### ISSUE TITLE
[Request] Automated Docker Hub build

### ISSUE THREAD

**Issue by kennethkalmer** [rlister/ecr-login#2]
Thanks so much for this!

I've just manually built and deployed ecr-login to a test CoreOS node and it is working great.

Could I recommend setting up an automated build on Docker Hub? Just to foster some trust that the ecr-login container published there is in fact generated purely from this repo?

**Comment by rlister** [comment]
@kennethkalmer I agree, good idea. I want to keep the image tiny (i.e. from `scratch`), so I need to figure out a way to setup the automated build with a binary.

**Comment by kennethkalmer** [comment]
That is a very good point, I didn't think that part through... Wonder if quay.io has the capabilities for this?

**Comment by jangrewe** [comment]
How about putting it in a ZIP here under releases, and then use ADD in your Dockerfile to download it?
Besides that: much appreciated, thank you! =)

**Comment by rlister** [comment]
@jangrewe I tried using `ADD` from releases, but docker will only uncompress _local_ archives; files from remote URLs are just copied. See https://docs.docker.com/engine/reference/builder/#add.

**Comment by kennethkalmer** [comment]
Does it need to be a ZIP? Looking at confd releases (https://github.com/kelseyhightower/confd/releases), Kelsey just puts the binaries up verbatim.

**Comment by jangrewe** [comment]
you're right (both), remote archives won't work =(
